htdocs/editicons.bml

```diff
@@ -117,32 +119,7 @@ use strict;
 
             my $MAX_UPLOAD = LJ::Userpic->max_allowed_bytes($u);
             my $error;
-
-            # act like the factory is disabled if no mogile
-            my $factory_disabled = !LJ::is_enabled('userpicfactory') || !LJ::mogclient();
```

Comment:
Yeah it was optional b/c of resources/setup costs (it required MogileFS) but BlobStore makes all file storage work (even in simple mode) so we can use those features even on Hacks. I figured with that true, it'd be worth simplifying/deleting the option.